tests/unit/Directors.spec.ts

```diff
@@ -736,152 +739,145 @@ describe('Appoint New Director tests', () => {
           }
       })))
 
-    const Constructor = Vue.extend(Directors)
-    const instance = new Constructor({ store, vuetify })
-    vm = instance.$mount()
-
-    // set props
-    vm.asOfDate = '2019-04-01'
-    vm.componentEnabled = true
+    wrapper = mount(Directors, {
+      sync: false,
```

Comment:
What does this sync property do in this case?

Reply:
It's a test framework "workaround" to allow things to happen synchronously, but it's admittedly "dangerous" because we're still using a beta version of the test-utils (and I think things break if we upgrade to the latest version, sigh).

Ref: https://github.com/vuejs/vue-test-utils/issues/1137

Reply:
Very interesting, thanks for clarifying!